spec/features/create_collection_spec.rb

```diff
@@ -18,51 +18,69 @@
     login_as user
   end
 
-  it 'creates a collection without creator or contributor' do
-    visit '/dashboard'
-    click_link "Collections"
-    click_link "New Collection"
+  context "when user is admin" do
+    it 'creates a collection without creator or contributor' do
+      user.add_role(:admin)
```

Comment:
I think this assumes the tests run in a certain order.  I think it would be better to add this to a `let` within the current context like:
`let(:user) { create(:admin)`
Although I wouldn't think this necessary if the feature is enabled by default because then it should appear for all users with dashboard access.

Reply:
Ahhh you might be right, this might be left over from when I had it off by default